### DISCUSSION TITLE
PhysicsShapeCache not found

### DISCUSSION THREAD

**Discussion by hasikill** [axmolengine/axmol#2059] [Q&A] [answered]
I ran the following example and found that there is no PhysicsShapeCache in the axmol API.
Example: https://www.codeandweb.com/physicseditor/tutorials/how-to-create-a-physics-enabled-game-with-axmol-engine
Is PhysicsEditor able to support it?

**Comment by rh101** [comment, marked as answer]
> I ran the following example and found that there is no PhysicsShapeCache in the axmol API.

The `PhysicsShapeCache` has nothing to do with Axmol.  It's created by the person who made that tutorial, and a github link to the project where the file exists is provided in that same article, so if you need it, grab it from there:

https://github.com/CodeAndWeb/PhysicsEditor-Loaders/tree/master/axmol

**Reply by hasikill** [reply]
I got it, thanks
